infrastructure/tooling/src/utils/command.js

```diff
@@ -42,7 +42,7 @@ exports.execCommand = async ({
       } else {
         output = '...\n' + chunk.toString();
       }
-      callback(null, output);
+      callback(null, chunk);
```

Comment:
huh, how did this even work before?

Reply:
It just output a lot of `...` to the stream, but if nothing went wrong you didn't really notice.